### ISSUE TITLE
Add option to choose which is main file and which needs to be merged

### ISSUE THREAD

**Issue by he2a** [Zweite93/Stalker2PakCfgMergeTool#7]
Currently the tool automatically decides it which leads to incorrect values being merged.

**Comment by STARasGAMES** [comment]
I would go even further and allow per-line resolve ability

**Comment by Hebutin** [comment]
so far i have used mix of both Simple Mod Manager that comes with Vortex mod manager and this, i use SMM to merge the files and then choose the manual option when it ask if i want to do 1 manual, 2 automatic and 3 skip.  then i use this after to make a second merge and i use the name this one makes and copy the name from to the SMMs pak.

i first had issues with SMMs pak since it still did not fix the conflict issues i had and after trying this one those mods started to work. then i tired using this managers pak name on the SMMs and then it started to work. 

in short. i ran the manual option with SMM to make sure it decides the right mods and changes > run this one to make a 2nd merge pak > copy this merge paks file name then delete it > rename the SMMs merge pak to this ones
